src/main/java/com/epam/bookstoreservice/dto/response/Result.java

```diff
@@ -71,5 +71,4 @@ public static  <T> Result<T> error(Integer errorCode, String message) {
         return new Result<>(errorCode, message, null);
     }
 
```

Comment:
You don't need to have extra code in your result body since the HTTP code can represent the state of the request.
You can remove the field 'code'.